api/node/rust/interpreter/component_compiler.rs

```diff
@@ -164,6 +167,68 @@ impl JsComponentCompiler {
             .collect::<HashMap<String, JsUnknown>>()
     }
 
+    #[napi(setter)]
+    pub fn set_file_loader(&mut self, env: Env, callback: JsFunction) -> napi::Result<()> {
+        let function_ref = RefCountedReference::new(&env, callback)?;
+
+        self.internal.set_file_loader(move |path| {
+            let Ok(callback) = function_ref.get::<JsFunction>() else {
+                return Box::pin(async {
```

Comment:
Instead of duplicating the Box::pin(async move { ... }) bits throughout the function, I think you can write one big async block and surround it by the pin.

Reply:
Why is that?